lib/Backend/Func.cpp

```diff
@@ -290,6 +290,47 @@ Func::IsLoopBodyInTryFinally() const
     return IsLoopBody() && m_workItem->GetLoopHeader()->isInTryFinally;
 }
 
+/* static */
+void
+Func::InsertObjectCheck(IR::RegOpnd * funcOpnd, IR::Instr* insertBeforeInstr,
```

Comment:
Since these are now only called from Lower.cpp, these functions probably belong in that file.